src/anchor/classes.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './index';
+
+.iui-anchor {
+  @include iui-anchor;
+
+  &-positive {
+    @include iui-anchor-status($status: positive);
+  }
+
+  &-warning {
+    @include iui-anchor-status($status: warning);
+  }
+
+  &-negative {
+    @include iui-anchor-status($status: negative);
+  }
```

Comment:
Not sure if we want to expose these to the users. It's one thing to make mixins for internal re-use, but if we provide these options then we might start seeing misuse

Reply:
Removed the status classes, we can test for those statuses within Alert and Toast.